Update EDM inputs to HLT-Validation tests [`10_3_X` only]

missirol opened the pull request:
#### PR description:

This PR is specific to the `CMSSW_10_3_X` release cycle.

Some of the HLT-Validation tests running in IBs started to fail due to the unavailability of EDM files used as input to these tests (see [IB dashboard](https://cmssdt.cern.ch/SDT/html/cmssdt-ib/#/ib/CMSSW_10_3_X), and https://github.com/cms-sw/cmssw/issues/40013).

This PR updates the path to these files, ~~to use copies kept in the TSG area on EOS.~~ which will now rely on the cms-bot cache (see discussion in https://github.com/cms-sw/cmssw/issues/40013 and #40020, where the corresponding updates to `cms-bot` are also mentioned).

In addition, it disables the HLT-integration-tests part of the HLT-Validation tests for scram-archs in SLC6, which are incompatible with the most recent version of ConfDB-v2 (see https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1326250032).

#### PR validation:

None.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR. If this PR will be backported, please specify to which release cycle the backport is meant for:

N/A

cmsbuild commented:
A new Pull Request was created by  @missirol (Marino Missiroli) for CMSSW_10_3_X.

It involves the following packages:

- Configuration/HLT (**hlt**)
- HLTrigger/Configuration (**hlt**)


@cmsbuild, @missirol, @Martin-Grunewald can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @silviodonato, @fabiocos this is something you requested to watch as well.
@perrotta, @dpiparo, @rappoccio you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

missirol commented:
please test

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-67d24c/28913/summary.html
**COMMIT**: e8b56e37b49668f8b1859c7a01e5a7549bc202ec
**CMSSW**: CMSSW_10_3_X_2022-11-06-0000/slc6_amd64_gcc700
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/40027/28913/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 2 differences found in the comparisons
 - DQMHistoTests: Total files compared: 32
 - DQMHistoTests: Total histograms compared: 3203312
 - DQMHistoTests: Total failures: 2
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3203113
 - DQMHistoTests: Total skipped: 197
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 31 files compared)
 - Checked 134 log files, 28 edm output root files, 32 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_3_X_2022-11-06-0000+67d24c/53998/triggerResults): no differences found

missirol committed:
Update EDM input files of HLT-Validation tests

cmsbuild commented:
Pull request #40027 was updated. @cmsbuild, @missirol, @Martin-Grunewald can you please check and sign again.

missirol commented:
please test

cmsbuild commented:
-1

**Failed Tests**: AddOn 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-67d24c/29216/summary.html
**COMMIT**: f710e08aa926f5c0dda9f3800b762c2381a3d61c
**CMSSW**: CMSSW_10_3_X_2022-11-20-0000/slc6_amd64_gcc700
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/40027/29216/install.sh` to create a dev area with all the needed externals and cmssw changes.


## AddOn Tests

- [fastsim](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-67d24c/29216/addOnTests/fastsim)
```
cmsDriver.py TTbar_8TeV_TuneCUETP8M1_cfi  --conditions auto:run1_mc --fast  -n 100 --eventcontent AODSIM,DQM --relval 100000,1000 -s GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,DIGI2RAW,L1Reco,RECO,EI,VALIDATION  --customise=HLTrigger/Configuration/CustomConfigs.L1THLT --datatier GEN-SIM-DIGI-RECO,DQMIO --beamspot Realistic8TeVCollision : FAILED - time: date Wed Nov 23 20:38:38 2022-date Wed Nov 23 20:33:36 2022 s - exit: 35584
```


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 4 differences found in the comparisons
 - DQMHistoTests: Total files compared: 32
 - DQMHistoTests: Total histograms compared: 3203312
 - DQMHistoTests: Total failures: 1
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3203114
 - DQMHistoTests: Total skipped: 197
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 31 files compared)
 - Checked 134 log files, 28 edm output root files, 32 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_3_X_2022-11-20-0000+67d24c/54276/triggerResults): no differences found

missirol commented:
Strange `addOnTest` failure: not seen in `10_3_X` IBs, not seen in tests of this PR 2 weeks ago, and cannot be related to this PR. Will rerun the tests, after clarifying if this PR should updated to address https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1325571973.

missirol committed:
Skip HLT-integration tests in SLC6

HLT-integration tests cannot run with SLC6 architectures,
due to an incompatibility with the latest .jar files of ConfDB-v2.
For further details, see https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1325571973

cmsbuild commented:
Pull request #40027 was updated. @cmsbuild, @missirol, @Martin-Grunewald can you please check and sign again.

missirol commented:
please test

See https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1326250032.

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-67d24c/29233/summary.html
**COMMIT**: 7f94efeda2b40b9461b208dcb8fbd5462c34e0bb
**CMSSW**: CMSSW_10_3_X_2022-11-20-0000/slc6_amd64_gcc700
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/40027/29233/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 32
 - DQMHistoTests: Total histograms compared: 3203312
 - DQMHistoTests: Total failures: 1
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3203114
 - DQMHistoTests: Total skipped: 197
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 31 files compared)
 - Checked 134 log files, 28 edm output root files, 32 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_3_X_2022-11-20-0000+67d24c/54291/triggerResults): no differences found

missirol commented:
+hlt

This PR fixes the HLT-Val tests currently failing in `10_3_X` IBs (access EDM files from cms-bot cache + disable HLT-integration tests in SLC6).

PS. The strange `fastsim` error (https://github.com/cms-sw/cmssw/pull/40027#issuecomment-1326059754) disappeared..

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_10_3_X IBs (tests are also fine) and once validation in the development release cycle CMSSW_12_6_X is complete. This pull request will now be reviewed by the release team before it's merged. @perrotta, @dpiparo, @rappoccio (and backports should be raised in the release meeting by the corresponding L2)

perrotta commented:
+1
